src/libsync/discoveryphase.cpp

```diff
@@ -366,6 +367,8 @@ static std::unique_ptr<csync_file_stat_t> propertyMapToFileStat(const QMap<QStri
                 // Piggy back on the persmission field
                 file_stat->remotePerm.setPermission(RemotePermissions::IsShared);
             }
+        } else if (property == "zsync") {
```

Comment:
If the property the server replies with is `zsync: true` this should make use of the value, otherwise `zsync: false` would have the wrong effect.

Reply:
The property is either there, or not at all, see:
https://github.com/owncloud/core/pull/29404/commits/7410e28f62ec1301ea733c592ae538306c0bc5d4#diff-7c0d9269290e245a3c8af0b0c654dd8cR149

Reply:
I see. So the property is something like `<oc:zsync/>`?

Reply:
I don't think I can have a property with no value based on what I saw on the server code, but like I linked, it's either there (with whatever value) or not at all ...

Reply:
I understand that you made the server reply `<oc:zsync>true</oc:zsync>` or nothing and if the server maintainers are fine with that, that's ok. But the implied semantics suggest that `<oc:zsync>false</oc:zsync>` will mean "no zsync available" and not "sure, use zsync".

Reply:
Server doesn't even support sending `<oc:zsync/>`, anyway I've put a check on value. But I still don't know why anyone would change *my* code on the server and not expect things to break ...